changelog/19616.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
Not sure if this qualifies as a feature or improvement. But features have a slightly different formatting. Will send link in PM

Reply:
+1 for `improvement` according to how this doc defines each categories:
https://hashicorp.atlassian.net/wiki/spaces/VAULT/pages/1311244491/Changelog+Process